ams/AssetAnalyzer.cs

```diff
@@ -30,24 +32,48 @@ internal class AssetAnalyzer : BaseMigrator
             _tracker = tracker;
         }
 
-        private async Task<AnalysisResult> AnalyzeAsync(MediaAssetResource asset, BlobServiceClient storage, CancellationToken cancellationToken)
+        private async Task<AnalysisResult> AnalyzeAsync<T>(T item, BlobServiceClient storage, CancellationToken cancellationToken)
```

Comment:
Not sure if it is the GitHub tool issue or not, I could not see any change that calls this AnalyzeAsync<T>( ) method for the storage account from the PR review page.